packages/baobab.rb

```diff
@@ -1,42 +1,41 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Baobab < Package
+class Baobab < Meson
   description 'Disk Usage Analyzer (also known as baobab) scans folders, devices or remote locations and and reports on the disk space consumed by each element.'
   homepage 'https://wiki.gnome.org/Apps/DiskUsageAnalyzer'
-  version '43.0'
+  version '45.0'
   license 'GPL-2+ and FDL-1.1+'
   compatibility 'x86_64 aarch64 armv7l'
-  source_url 'https://gitlab.gnome.org/GNOME/baobab/-/archive/43.0/baobab-43.0.tar.bz2'
-  source_sha256 'dd1e2c65d4cee3d904d1c127ef700d07b8011a689cde2c3f9de04cdaa1f1df6c'
+  source_url 'https://gitlab.gnome.org/GNOME/baobab.git'
+  git_hashtag version
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: 'bd42ba8dd8345a57b294eed20b6eed797c1a2c75fe87d86640548c8879fef8f2',
-     armv7l: 'bd42ba8dd8345a57b294eed20b6eed797c1a2c75fe87d86640548c8879fef8f2',
-     x86_64: 'ee45c716382ad00b465ee704f10f266712239d7426098cdeb780f7a48a57b5e8'
+    aarch64: '7405ce4a84ca2ffeab17643a9efa11ced62a1687ee3c8433215bda8f177b541f',
+     armv7l: '7405ce4a84ca2ffeab17643a9efa11ced62a1687ee3c8433215bda8f177b541f',
+     x86_64: '0e1c62920726909f86756309f9a1a2efc7a520eaa78c63d88678bdbd4ea11e67'
   })
 
-  depends_on 'gtk3'
-  depends_on 'gsettings_desktop_schemas'
+  depends_on 'desktop_file_utils' => :build
+  depends_on 'gtk3' => :build
+  depends_on 'gsettings_desktop_schemas' => :build
   depends_on 'itstool' => :build
   depends_on 'vala' => :build
   depends_on 'vulkan_headers' => :build
-  depends_on 'vulkan_icd_loader'
+  depends_on 'vulkan_icd_loader' => :build
+  depends_on 'glib' => :build
+  depends_on 'glibc' => :build
+  depends_on 'harfbuzz' => :build
+  depends_on 'pango' => :build
+  depends_on 'gtk4' => :build
+  depends_on 'libadwaita' => :build
+  depends_on 'cairo' # R
+  depends_on 'harfbuzz' # R
   depends_on 'glib' # R
   depends_on 'glibc' # R
-  depends_on 'harfbuzz' # R
-  depends_on 'pango' # R
   depends_on 'gtk4' # R
```

Comment:
Couple things-- these dependencies arent alphabetical, and we have `gtk4` twice (and also `gtk3`? does it actually use both?)

I'm working on something to lint stuff ilke this, but a manual review every so often will suffice in the meantime.

Reply:
Fixed this file.  I tend to change all the dependencies to `=> :build` during a build and then run the getrealdeps script to see if any of those are actual runtime deps.

Reply:
(Forgot to sort and cull after that step.)